virtualization/windowscontainers/container-networking/network-drivers-topologies.md

```diff
@@ -13,14 +13,14 @@ ms.assetid: 538871ba-d02e-47d3-a3bf-25cda4a40965
 
 # Windows Container Network Drivers  
 
-In addition to leveraging the default 'nat' network created by Docker on Windows, users can define custom container networks. User-defined networks can be created using using the Docker CLI, [`docker network create -d <NETWORK DRIVER TYPE> <NAME>`](https://docs.docker.com/engine/reference/commandline/network_create/), command. On Windows, the following network driver types are available:
+In addition to leveraging the default 'nat' network created by Docker on Windows, users can define custom container networks. User-defined networks can be created using using the Docker CLI [`docker network create -d <NETWORK DRIVER TYPE> <NAME>`](https://docs.docker.com/engine/reference/commandline/network_create/) command. On Windows, the following network driver types are available:
 
 - **nat** – containers attached to a network created with the 'nat' driver will receive an IP address from the user-specified (``--subnet``) IP prefix. Port forwarding / mapping from the container host to container endpoints is supported.

```

Comment:
Please add, "will be connected to an internal Hyper-V switch" after "..."created with the 'nat' driver" so that it reads "containers attached to a network created with the 'nat' driver will be connected to an internal Hyper-V switch and will receive an ip address..."